cognite_toolkit/_cdf_tk/prototypes/commands/dump_timeseries.py

```diff
@@ -0,0 +1,306 @@
+            else:
+                selected.append("No data set selected.")
+            selected_str = "\n".join(selected)
+            what = questionary.select(
+                f"\n{selected_str}\nSelect a data set to dump",
+                choices=[
+                    "Data Set",
+                    "Done",
+                ],
+            ).ask()
+
+            if what == "Done":
+                break
+            elif what == "Data Set":
+                _available_data_sets = self._get_available_data_sets(client)
+                selected_data_set = questionary.checkbox(
+                    "Select a data set",
+                    choices=sorted(item for item in _available_data_sets if item not in data_sets),
+                ).ask()
+                if selected_data_set:
+                    data_sets.update(selected_data_set)
+                else:
+                    print("No data set selected.")
+        return list(data_sets)
+
+    def _get_available_data_sets(self, client: CogniteClient) -> set[str]:
+        if self._available_data_sets is None:
+            self.data_set_by_id.update({item.id: item.as_write() for item in client.data_sets})
+            self._available_data_sets = {item.external_id for item in self.data_set_by_id.values() if item.external_id}
+        return self._available_data_sets
+
+    def _to_write(
+        self,
+        time_series: Iterator[TimeSeriesList],
+        client: CogniteClient,
+        expand_metadata: bool,
+    ) -> Iterator[list[Any]]:
+        for time_series_list in time_series:
+            write_time_series: list[dict[str, Any]] = []
+            # TODO: how to separate `times_series` as list from `time_series` as one item?
+            for one_time_series in time_series_list:
+                write = one_time_series.as_write().dump(camel_case=True)
+                write.pop("parentId", None)
+                if "dataSetId" in write:
+                    data_set_id = write.pop("dataSetId")
+                    self._used_data_sets.add(data_set_id)
+                    write["dataSetExternalId"] = self._get_data_set_external_id(client, data_set_id)
+                if "assetId" in write:
+                    asset_id = write.pop("assetId")
+                    self._used_assets.add(asset_id)
+                    write["assetExternalId"] = self._get_asset_external_id(client, asset_id)
+                if expand_metadata and "metadata" in write:
+                    metadata = write.pop("metadata")
+                    for key, value in metadata.items():
+                        write[f"metadata.{key}"] = value
+                if "rootId" in write:
+                    root_id = write.pop("rootId")
+                    write["rootExternalId"] = self._get_time_series_external_id(client, root_id)
+                write_time_series.append(write)
+            yield write_time_series
+
+    def _get_time_series_external_id(self, client: CogniteClient, root_id: int) -> str:
```

Comment:
I would call the parameter just `time_series_id` 
```suggestion
    def _get_time_series_external_id(self, client: CogniteClient, time_series_id: int) -> str:
```

Reply:
Based on the comment above, not sure if this function is needed at all.